crates/cast/src/starknet_commands/account/add.rs

```diff
@@ -37,6 +54,10 @@ pub struct Add {
     /// If passed, a profile with corresponding data will be created in Scarb.toml
     #[clap(long)]
     pub add_profile: bool,
+
+    /// If passed, import account from keystore and starkli account JSON file
+    #[clap(short, long, conflicts_with_all = ["address", "class_hash", "deployed", "private_key", "public_key", "salt"])]
```

Comment:
`conflicts_with` makes it not possible to pass from_keystore with any of the arguments

Reply:
why not just do whatever is done with this flag with just `--keystore` argument?